src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValueNewTable.cs

```diff
@@ -29,7 +29,7 @@ public partial class FormulaValue
         /// <returns></returns>
         public static TableValue NewTable(RecordType recordType, params RecordValue[] values)
         {
-            return NewTable(recordType, (IEnumerable<RecordValue>)values);
+            return NewTable(recordType, values.ToList());
```

Comment:
>values.ToList() [](http://example.com/codeflow?start=40&length=15)

This is subtle - 
if we pass in an array, we're making a copy into a mutable list.   So the return value will always be mutable. 

whereas if we call the overload on Line 35, we avoid copying, but instead querying for a mutable interface. and it's potentially not mutable. 

Not necessarily wrong, just raising awareness of the semantics. Good for now, we'll likely refine further.